Deploying_vLLM/template_vllm_server_fp8.sh

```diff
@@ -6,20 +6,20 @@ unset VLLM_PROMPT_SEQ_BUCKET_MAX
 MODEL_BASE=$(echo $MODEL | awk -F '/' '{print $2}')
 MODEL_BASE=${${q}MODEL_BASE,,}
 
-if [ -f "./measurement/${${q}MODEL_BASE}_tp${${q}TENSOR_PARALLEL_SIZE}_${${q}gnum}/${${q}MODEL_BASE}/maxabs_quant_$gnum.json" ]; then
+if [ -f "./measurement/${${q}MODEL_BASE}_tp${${q}TENSOR_PARALLEL_SIZE}_${${q}gnum}_${${q}VER}/${${q}MODEL_BASE}/maxabs_quant_$gnum.json" ]; then
 	echo "Measurement file found, skipping calibration"
 else
 	cd /root/scripts/vllm_hpu_ext/calibration/
-	echo "UNI_GROUPS is: [$UNI_GROUPS]"
+	echo "RANK is: $RANK"
 
-	if [ -n "$UNI_GROUPS" ] && [ "$UNI_GROUPS" != "None" ]; then
+	if [ -n "$RANK" ] && [ "$RANK" != "None" ]; then
         	echo -e 'Calibrate model with unification'
-	        ./calibrate_model.sh -m $MODEL -d /root/scripts/dataset-processed.pkl -o /root/scripts/measurement/${${q}MODEL_BASE}_tp${${q}TENSOR_PARALLEL_SIZE}_${${q}gnum} -l 100 -t $MEASUREMENT_TP -g "$UNI_GROUPS"
+	        ./calibrate_model.sh -m $MODEL -d /root/scripts/dataset-processed.pkl -o /root/scripts/measurement/${${q}MODEL_BASE}_tp${${q}TENSOR_PARALLEL_SIZE}_${${q}gnum}_${${q}VER} -l 100 -t $TENSOR_PARALLEL_SIZE -r $RANK
```

Comment:
Have you tested this? Seems like some logic error, -r should be TENSOR_PARALLEL_SIZE and -t MEASUREMENT_TP